### ISSUE TITLE
Does this project actually use pytz?

### ISSUE THREAD

**Issue by matmair** [timvink/mkdocs-git-revision-date-localized-plugin#154]
Hi! The requirements list pytz but I do not see references in the code. Do you know if it is still required? Python contains [zoneinfo](https://docs.python.org/3/library/zoneinfo.html) starting with 3.9 so I am trying to get pytz out of our dependency tree.

**Comment by timvink** [comment]
Good point, but then we have to drop support for 3.8, which is still supported by mkdocs.

Maybe you can fork the project until mkdocs drops support for 3.8 (then we can update this project also).

**Comment by matmair** [comment]
@timvink already have, just wanted to check if upstream was aware that dropping this was possible in the future. Thank you for the great package, this just works and that is very awesome!

**Comment by timvink** [comment]
Planned in https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/issues/158

**Comment by timvink** [comment]
I'll make a release tomorrow with the change :)

https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/pull/193
